### PR TITLE
chore: Update Billing and Usage docs with AWS Marketplace info

### PR DESCRIPTION
**Pull request by JoeKarlsson**
I updated the CloudQuery documentation to include information about purchasing CloudQuery through the AWS marketplace. 

I also made some minor updates to grammar and copy to improve clarity.

### CODE REVIEW

File: website/pages/docs/billing-and-usage/overview.md [hunk 1]
```diff
@@ -7,30 +7,34 @@ description: Learn about CloudQuery's billing model
...
 ![Usage chart](/images/docs/billing-and-usage/usage-chart.png)
 
-If you used cloud syncs, your egress, vCPU, and vRAM consumption will be displayed at the bottom of the page.
+If you use cloud syncs, your egress, vCPU, and vRAM consumption will be displayed at the bottom of the page.
 
 ![Cloud syncs usage](/images/docs/billing-and-usage/cloud-syncs.png)
+
+## AWS Marketplace
+
+You can also purchase CloudQuery directly through the [AWS Marketplace](https://aws.amazon.com/marketplace/pp/prodview-lowyuyay5a37s#pdp-pricing). This option streamlines billing and usage management, seamlessly integrating CloudQuery into your existing AWS environment. Explore the pricing and get started with CloudQuery on the AWS Marketplace today.
```
Comment (bbernays) [inline website/pages/docs/billing-and-usage/overview.md:40]: The usage management within the AWS Marketplace is not a feature we want to highlight. It is super difficult to understand usage in the AWS marketplace. The Cost & Explorer Explorer that is intended for understanding usage takes hours to update.

**Comment by marianogappa** [comment]
I read the changes and LGTM 👍 I'm gonna let @bbernays approve after his comment is addressed.

File: website/pages/docs/billing-and-usage/overview.md (hunk 1 above)
Comment (JoeKarlsson) [reply website/pages/docs/billing-and-usage/overview.md:40]: I hear you - we do not want to position it as the best way to use CQ. I added it to the bottom of the billing section, but how would you feel if I softened the language a bit? I still think it's important to mention that the AWS marketplace is an option. Something short and simple like this:

`You can also purchase CloudQuery directly through the [AWS Marketplace](https://aws.amazon.com/marketplace/pp/prodview-lowyuyay5a37s#pdp-pricing).`

Comment (bbernays) [reply website/pages/docs/billing-and-usage/overview.md]: Yeah that looks good!

**Review by bbernays — APPROVED** [review]

**Pull request merged**